content/news/2023-05-18-maplibre-newsletter-may-2023/index.md

```diff
@@ -0,0 +1,89 @@
+
+- Lazy load default style properties on demand to improve loading performance and reduce memory usage: [#2476](https://github.com/maplibre/maplibre-gl-js/pull/2476)
+- Remove support for `hsl` CSS color in a format that does not comply with the CSS color specification. Colors defined in the following format **will no longer work**: `hsl(110, 0.7, 0.055)`. Instead **it is recommended** to use the percentage format: `hsl(110, 70%, 5.5%)`. See the PR: [#2376](https://github.com/maplibre/maplibre-gl-js/pull/2376)
+- Remove "mapbox-gl-supported" package from API. If needed, please reference it directly instead of going through MapLibre: ([#2451](https://github.com/maplibre/maplibre-gl-js/pull/2451))
+- Remove WebGL1 support: ([#2512](https://github.com/maplibre/maplibre-gl-js/pull/2512))
+
+## MapLibre Native
+
+People interested in contributing can schedule a call with Bart Louwers, Maintainer MapLibre Native: <a href="https://github.com/maplibre/maplibre-native/discussions/898">maplibre/maplibre-native#898</a>.
+
+#### Contributions
+
+Most notable contributions since April 14th, 2023:
+
+- "maplibre-gl-native" became "maplibre-native": [#1090](https://github.com/maplibre/maplibre-native/pull/1090)
+- Code format via clang-format was added: [#1086](https://github.com/maplibre/maplibre-native/pull/1086)
+- Code coverage can now be checked [here](https://app.codecov.io/github/maplibre/maplibre-native/tree/main)
+- Render tests are now running on Linux. The test results are uploaded and a bot posts a link to the results on a PR: [#1069](https://github.com/maplibre/maplibre-native/pull/1069)
+- A size test was created for iOS. Similar as above, a bot will comment on each PR regarding the percentage size change of the library: [#1092](https://github.com/maplibre/maplibre-native/pull/1092)
+
+## Metal
+
+Check up on weekly updates from now on [here](https://github.com/maplibre/maplibre-native/discussions/1072) and follow the project roadmap [here](https://github.com/orgs/maplibre/projects/8)
+
+Highlights since April 14th, 2023:
+
+- Restructuring the code around drawables, builders and render loop
+- Introducing [UBOs](https://www.khronos.org/opengl/wiki/Uniform_Buffer_Object)
+- Refactoring layers starting with BackgroundLayer, FillLayer, CircleLayer and LineLayer
+
+## Events
+
+We would like to point out the [MapLibre User Group Japan](https://bit.ly/mug--jp) initiated by [Yasunori](https://github.com/dayjournal). The 5th event will take place [here](https://twitter.com/i/spaces/1nAJErzVBvgxL), June 7th, 2023 19:00 (JST). At the previous event 32 listeners participated when [MapLibre Native](https://github.com/maplibre/maplibre-native) was introduced. Another focus was [FOSS4G](https://2023.foss4g.org/). Anyone who is interested, please do not hesitate to join. Everyone is welcome.
+
+## Next Up
+
+We are looking forward to the preparations on the upcoming Governing Board Elections that will take place August 24th, 2023.
```

Comment:
Please add a link to the MapLibre talks at SotMUS. @nyurik  is presenting there, also a few more. Please check in the slack channel, I think @roblabs posted all MapLibre talks there...

Reply:
Thanks @wipfli 

Please use this link for State of the Map
* https://2023.stateofthemap.us/schedule 

Not necessarily all talks listed in Slack are MapLibre related.  This list is those that are with members in #MapLibre OSM Slack channel
* Cross-referenced 2023 schedule announcement with members in #MapLibre OSM Slack channel — https://osmus.slack.com/archives/C01G3D28DAB/p1682614633469129